src/components/CustomerDetails/InputIdSection.tsx

```diff
@@ -39,18 +39,20 @@ const styles = StyleSheet.create({
   }
 });
 
-interface InputNricSection {
+interface InputIdSection {
```

Comment:
Hmm should we name the interface differently? Currently the function component below and this interface have the same name haha

Reply:
hmm, looking thru our codebase, this seems like this is the usual practice, maybe @sebastianquek was there any reason why this was the case? i.e. interface and function having identical naming

Reply:
Yea good question, I took reference from somewhere sometime back, but I can't remember where. Some considerations:
1. There's little use in differentiating between the interface and function for React function components.
Typescript and our IDEs already help to resolve whether we want to use a name as an interface or function. For example, let's say we export this `InputIdSection` interface as well. In other modules, we can do the following and everything works fine:
```ts
import { InputIdSection } from "./InputIdSection"; // Just need 1 import
...

interface ExtendedInputIdSection extends InputIdSection { // If the `InputIdSection` is not exported, then we'll immediately get a warning
  newProp: boolean;
}
...

const SomeComponent: FunctionComponent = () => (
  <InputIdSection ... /> // Resolved nicely here
)

```

2. We could add `Props` to the back of these interfaces, but those are additional constraints that we need to enforce. Right now we can do either an inline type, extracted type or extracted interface. So far this flexibility hasn't caused issues and the general rule of thumb is that for small functions with few props, just do it as an inline type. Am open to enforcing more constraints, but if we cannot automate and detect it, may be more of a bane.

Reply:
I see I see~ oooh is it this? 
https://www.typescriptlang.org/docs/handbook/declaration-merging.html

Reply:
I think it's less about merging more of the name referring to different entities, perhaps more about polymorphism?